### ISSUE TITLE
[$10][MSFT-248] Sample challenge page: Challenge prerequisites popup: Unlabelled button at the end of the popup.

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2901]
**Steps to Reproduce**
1. Go to the Sample challenge page: https://www.topcoder.com/challenges/30095347. 2. Ensure that the screen reader is switched on. 3. activate the register button so that the challenge prerequisites popup is displayed. 4. Press the key combination 'ctrl + insert + b' to bring up a list of buttons on the page. 5. Note there is an unlabelled button at the end of this list. 6. This button when activated closes the challenge prerequisites popup.

**Expected Result**
Screen reader should announce the appropriate label for the button.

**Actual Result**
Screen reader reads an unlabelled button at the end of the challenge prerequisites popup which when activated closes the popup.

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 1. Perceivable
**WCAG Level**: A
**WCAG Success Criterion**: 1.3.1 Info and Relationships
**Screen Reader Used**: JAWS
**Accessibility Audit Tool used**: N/A

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=088f5f5d%2D9b1c%2D4b7e%2Dacbf%2D320c257c9a9a&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2FKameshwarid%2Fbug%2042

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30095983 has been created for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30095983 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30095983 has been updated - it has been assigned to gets0ul.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2901 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2901 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2901 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096747 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096747 has been updated - it has been assigned to gets0ul.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by sachin-maheshwari** [comment]
Unable to test due to dependency.

**Comment by crazyk07** [comment]
Closing ticket due to shift in priority, will not be able to complete but will still pay for effort rendered.

Note: Ticket was dependent on react Utils

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30096747<br/><br/>```This is an automated message for crazyk via Topcoder X```
